project/Build.scala

```diff
@@ -12,9 +12,9 @@ import com.typesafe.sbt.sdlc.Plugin._
 
 object SlickBuild extends Build {
 
-  val slickVersion = "2.1.0"
+  val slickVersion = "2.1.0-Scala2.12"
```

Comment:
I think this should be set to 2.1.1-SNAPSHOT. We still checked release versions into git before we had tag-driven releases, so the next PR to a branch should switch it back to a snapshot version.

Reply:
Fixed